src/pytype.jl

```diff
@@ -280,7 +280,7 @@ type PyTypeObject
              Py_TPFLAGS_HAVE_CLASS |
              Py_TPFLAGS_HAVE_STACKLESS_EXTENSION |
              Py_TPFLAGS_HAVE_INDEX)
-        name_save = bytestring(name)
+        name_save = Base.cconvert(Ptr{UInt8}, name)
```

Comment:
This is the `ccall` interface. The code here is exactly the same as if you write `ccall(..., (..., Ptr{UInt8}, ...,), ..., name, ...)`